src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -307,7 +307,21 @@
         ],
         "Temperature Control": ["SupportedTemperatureLevels"],
         "Operational State": ["OperationalState", "OperationalError"],
-        "RVC Operational State": ["OperationalState", "OperationalError"]
+        "RVC Operational State": ["OperationalState", "OperationalError"],
+        "Activated Carbon Filter Monitoring": [
+            "Condition",
```

Comment:
FeatureMap too, no? And for the other monitoring cluster and the other json file.

Reply:
done